install.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
it's not using any bash specifics

```suggestion
#!/bin/sh
```